Cargo.lock

```diff
@@ -204,7 +204,7 @@ checksum = "b5a972e5669d67ba988ce3dc826706fb0a8b01471c088cb0b6110b805cc36aed"
 
 [[package]]
 name = "vtebench"
-version = "0.3.0"
+version = "0.3.1"
```

Comment:
I didn't update the lockfile in the previous commit, which is the main reason for this bump. I've pushed to crates.io despite this, but that shouldn't be a problem I think.